source/includes/_dataset_actions.md

```diff
@@ -0,0 +1,152 @@
+# Dataset actions
+
+A state of a dataset can be changed by triggering actions. An action can make the dataset available through the search API or remove it from the search API.
+
+
+## Publish a dataset
+
+> Definition
+
+```http
+PUT https://{DOMAIN_ID}.opendatasoft.com/api/management/v2/datasets/{DATASET_UID}/publish
+```
+
+> Example request
+
+```HTTP
```

Comment:
Maybe, but that's how we did in all the other parts of the doc.
It ends in under the shell tab.

I don't know why, but we may want to do a global refactoring.